.github/dependabot.yml

```diff
@@ -2,16 +2,186 @@
 
 version: 2
 updates:
-  # Enable version updates for Python API
-  - package-ecosystem: pip
-    directory: "/src/bindings/python"
-    schedule:
-      interval: weekly
-      day: sunday
-      time: "13:00"
-    open-pull-requests-limit: 0
-    reviewers:
-      - jiwaszki
-      - akuporos
-    labels:
-      - "category: dependencies"
+  #
+  # Python product dependencies
+  #
+
+  # Python API requirements
+  - package-ecosystem: pip
+    directory: "/src/bindings/python/"
+    schedule:
+      interval: "daily"
+      time: "09:00"
+      timezone: "Poland"
+    open-pull-requests-limit: 3
+    assignees:
```

Comment:
Every `assignees` field related to Python probably should also include @akuporos.

Reply:
`reviewers` you mention are already assigned via CODEOWNERS file. See https://github.com/orgs/openvinotoolkit/teams/openvino-configuration-mgmt/members